views/artist-search-results.hbs

```diff
@@ -0,0 +1,10 @@
+  {{#each artists}}
+
+    <div >
+        <h2 class="name">{{name}}</h2>
+        <img src="{{images.0.url}}" alt="{{name}}">
```

Comment:
There might be no image.
Hence, sometimes no image is displayed.
You could fix this similarly to the way you fixed that issue on line 45 in app.js.